pyproject.toml

```diff
@@ -29,9 +29,9 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Artificial Intelligence",
 ]
 dependencies = [
-    # "optimum~=1.26",
+    # "optimum~=1.27",
     "optimum @ git+https://github.com/huggingface/optimum.git@mv-onnx",
-    "transformers>=4.36,<4.53.0",
```

Comment:
For my knowledge: what is the policy for supporting transformers versions? When can be drop support for an old version?